src/components/crud/create.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable  import/no-cycle, import/no-mutable-exports, import/prefer-default-export */
```

Comment:
Please make sure you follow the [YAGNI](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/clean-code/lesson_DRY_KISS_YAGNI.md). 
YAGNI: You Aren't Gonna Need It. Don't add anything you don't strictly need. Try to avoid the temptation of adding the most trendy technologies just because you think they may be useful in the future. Add things gradually, when they are really needed.